test/machinesxmls.py

```diff
@@ -100,7 +100,7 @@
   <bridge name='virbr0' stp='on' delay='0'/>
   <mac address='52:54:00:53:7d:8e'/>
   <ip address='192.168.122.1' netmask='255.255.255.0'>
-    <tftp root='/var/lib/libvirt/pxe-config'/>
+    <tftp root='/var/lib/libvirt/dnsmasq'/>
```

Comment:
Ah, so dnsmasq is also a PXE and TFTP server? Then this makes sense.

Reply:
Yes, see https://dnsmasq.org/doc.html